test/Microsoft.Azure.ServiceBus.UnitTests/ExpectedMessagingExceptionTests.cs

```diff
@@ -48,66 +46,11 @@ public class ExpectedMessagingExceptionTests
             }
         }
 
-        // TODO: Update this when Session is implemented
-        /*
-        [Fact]
-        async Task SessionLockLostExceptionTest()
```

Comment:
revert this. Let it stay commented for now. We need to change this to work with receiver instead of queueClient